index.html

```diff
@@ -34,47 +35,79 @@
             </recipes>
 
             <main class="main-container">
-                <maintext class="box">
+                <main-text class="box">
                     <h1>Anti-Xeno Synthesis Calculator</h1> 
 
-                    <input type="text" id="heatsink" placeholder="0">
+                    <button1 class="clear-input" onclick="clearInput(heatsink)"><span class="material-icons">block</span></button1>

```

Comment:
you don't define a "clear-input" css class. It would be better if you just used the standard button tag (rather than a whole new one) and then applied the styling changes you want using classes.

Reply:
fixed